launch/dynamo-tui/docker-test.sh

```
@@ -0,0 +1,31 @@
+fi
+
+echo "Running dynamo-tui tests in Docker..."
+echo ""
+
+# Run tests (build if needed, then test)
+docker run --rm \
+    -v "$PROJECT_ROOT:/workspace" \
+    -w /workspace \
+    rust:1.90.0-slim bash -c "
+        apt-get update -qq && apt-get install -y -qq pkg-config libssl-dev build-essential protobuf-compiler >/dev/null 2>&1
+        cargo test -p dynamo-tui -- '$@'
```

Shellcheck (0.11.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)